_studio/mfx_lib/decode/vp8/src/mfx_vp8_dec_decode_hw.cpp

```diff
@@ -454,6 +454,29 @@ mfxStatus VideoDECODEVP8_HW::GetOutputSurface(mfxFrameSurface1 **pp_surface_out,
     return MFX_ERR_NONE;
 }
 
+mfxStatus VideoDECODEVP8_HW::SetFrameType(mfxFrameSurface1 *p_surface_out)
```

Comment:
@dmitry-gurulev : would that better be `mfxFrameSurface1&` rather than `mfxFrameSurface1*`?

Reply:
@dvrogozh, yes it would

@akokoshn, what about
`auto extFrameInfo = static_cast<mfxExtDecodedFrameInfo*>(GetExtendedBuffer ...)`
?